hypervisor/src/vm.rs

```diff
@@ -390,6 +395,17 @@ pub trait Vm: Send + Sync + Any {
     ) -> Result<()> {
         unimplemented!()
     }
+
+    /// Freeze the Partiton/VM
```

Comment:
I would prefer reuse the `pause` / `resume` terminology. There is not need for the `_vm` suffix as this is on the Vm trait.

Please also make it unconditional at build time and simply provide an empty implementation in the trait.

Reply:
Are you refering to rename the function name or implementing the pausable trait? I skipped implementing the pauasable trait it it requires mutable reference to vm. I can easily rename the functions and make it unconditional. Let me know. Thanks

Reply:
Just the name - I don't expect you to implement the trait.